src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { ApplicationConfig } from '@angular/core';
+import { provideRouter } from '@angular/router';
+
+import { routes } from './app.routes';
+
```

Comment:
suggestion

          providers: [provideRouter(routes, { enableTracing: true })]

          Explantion: Enabling tracing for the router can help debug routing issues by logging routes as they are activated.